README.md

```diff
@@ -1,7 +1,7 @@
 [![Version](https://img.shields.io/github/package-json/v/0xProject/0x-api)](https://github.com/0xProject/0x-api/releases)
 [![Docs](https://img.shields.io/badge/docs-website-yellow.svg)](https://0x.org/docs/api)
 [![Chat with us on Discord](https://img.shields.io/badge/chat-Discord-blueViolet.svg)](https://discord.com/invite/d3FTX3M)
-[![Continuous Integration](https://github.com/0xProject/0x-api/workflows/Build%20and%20Test/badge.svg)](https://circleci.com/gh/0xProject/0x-api/tree/master)
+[![Continuous Integration](https://github.com/0xProject/0x-api/workflows/Build%20and%20Test/badge.svg)](https://github.com/0xProject/0x-api/actions?query=workflow%3A%22Build+and+Test%22+branch%3Amaster)
```

Comment:
dang, I changed the badge but forgot to change the actual link! thanks for catching this.

https://github.com/0xProject/0x-api/pull/213/files#diff-04c6e90faac2675aa89e2176d2eec7d8R4

Reply:
Yea honestly I just needed a diff to get rid of the circle CI "failed" build and found this :P